Eastern-Research-Group/expert-query
# Feature/191 organize fields

## Related Issues:
* [EQ-191](https://jira.epa.gov/browse/EQ-191)
* [EQ-195](https://jira.epa.gov/browse/EQ-195)
* [EQ-196](https://jira.epa.gov/browse/EQ-196)
* [EQ-197](https://jira.epa.gov/browse/EQ-197)
* [EQ-199](https://jira.epa.gov/browse/EQ-199)
* [EQ-200](https://jira.epa.gov/browse/EQ-200)
* [EQ-201](https://jira.epa.gov/browse/EQ-201)

## Main Changes:
* Grouped filter fields into logical categories, which vary depending on the selected profile.
  * Each grouping is still laid out in flexible grid rows, from left-to-right and top-to-bottom. The EPA team may decide to use a "column-first" layout and/or to create other sub-groupings, but I feel that this is a good starting point.
* Added the "Action Type" field to the _Actions_ profile, which had been previously missed.
* Assigned `Active` as the default value of the "Assessment Unit Status" field.

## Steps To Test:
1. Navigate to http://localhost:3000/attains/assessments.
2. Confirm that `Active` is selected as the default "Assessment Unit Status" value.
3. Review the categorization of fields in each profile, and verify that they match the specifications in the following document (the "Organization Type" field in all profiles and the "Fiscal Year" field in the _Actions_ profile are awaiting further discussion):
[profile-layouts.pdf](https://github.com/Eastern-Research-Group/expert-query/files/11151389/profile-layouts.pdf)
